assets/src/domain/shared/entities/datetimes/helpers/statusBgColor.ts

```diff
@@ -0,0 +1,19 @@
+import { Datetime } from '@edtrServices/apollo/types';
+
+const statusBgColor = (date: Datetime): string => {
```

Comment:
Is this returning class name ?
If so, `statusBgColorClassName` would be more explicit

Reply:
agreed! will do